calplus/conf/providers.py

```diff
@@ -29,23 +29,25 @@
                default=''),
     cfg.StrOpt('os_novaclient_version',
                default='2.1'),
+    cfg.StrOpt('os_swiftclient_version',
+               default='2'),
```

Comment:
Oh shit, i use wrong term here. this is not `os_swiftclient_version`. It should be `auth_version` - OpenStack auth version. Not like python-novaclient, python-swiftclient doesn't have several versions. 
I'll fix it ASAP.